app/code/local/Mowdirect/Emailimporter/Helper/GmailConnect.php

```diff
@@ -1,9 +1,13 @@
 <?php
 
+ini_set('display_errors', 1);
+error_reporting(E_ALL);
+
 class Mowdirect_Emailimporter_Helper_GmailConnect extends Mage_Core_Helper_Abstract {
 
     private $token;
     private $client;
+    private $redirectUri = 'http://127.0.0.1/index.php/email_importer/index/gmailredirect/';
```

Comment:
@sudarshann The http://127.0.0.1 URL isn't going to work with kubernetes use something similar to this https://stackoverflow.com/questions/11450146/magento-generating-url-for-a-backend-action-with-key